README.md

```diff
@@ -367,6 +368,45 @@ $ kubectl get node node1 -o json | jq '.status.allocatable'
 
 ```
 
+### Virtual environments with no iommu
```

Comment:
This seems to me  a rather specialized usecase, IMO we move this section under docs/dpdk/README-virt

we can have a general description here and point to docs/dpdk/README-virt
we should also explain that not all selectors are applicable.

e.g

### Virtual environments

SR-IOV network device plugin supports running in a virtualized environment, however, not all device selectors are applicable as the VFs are passthrough to the VM without any association to their respective PF, hence any device selector that relies on the association between a VF and its PF will not work.

The following selectors will not work in a virtualized environment: `pfNames`, `rootDevices`

#### Virtual environments with no iommu
SR-IOV network device plugin supports allocating VFIO devices in a virtualized environment without a virtualized iommu
for more information refer to *{add link}*

Reply:
P.S
I realize we are missing some documentation on consuming sr-iov resources in virtualized environment in general, however thats not related to this PR.